guides/common/attributes.adoc

```diff
@@ -15,6 +15,8 @@
 
 // Use current RH Satellite release version for links to access.redhat.com
 :AccessRedHatComVersion: 6.10-beta
+// TODO: Should this be removed and really always be BaseURL?
+:AccessMultiUrl: https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html/
```

Comment:
Isn't this exactly what we have here:

```
$ ag :BaseURL: -A2 -B1
guides/common/attributes.adoc
51-:SiteURL: https://docs.theforeman.org
52::BaseURL:  {SiteURL}/{ProjectVersion}/
53-include::attributes-base.adoc[]
54-
--
68-ifdef::satellite[]
69::BaseURL: https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html-single/
70-include::attributes-satellite.adoc[]
71-endif::[]
--
73-ifdef::orcharhino[]
74::BaseURL: https://docs.orcharhino.com/
75-include::attributes-orcharhino.adoc[]
76-endif::[]
```

Reply:
Notice it's `/html/` vs `/html-single/`. I can't judge if we can align on one or the other though I'd strongly prefer that.

Reply:
Did you check if this works with html? I don't think it would because html link requires linking a section and then a subsection. I think BaseURL is doing what you are trying to do here.

Reply:
Note I'm not using it everywhere, just replaced the hardcoded substring with a variable. All links I'm replacing were already `html` so if it's broken, it was broken before.